packages/core/src/__tests__/caip10-link.test.ts

```diff
@@ -94,7 +108,7 @@ describe('Ceramic API', () => {
     })
 
     it('Create and link DID', async () => {
-      const account = '0x0544DcF4fcE959C6C4F3b7530190cB5E1BD67Cb7@eip155:1'
+      const account = 'eip155:1:0x0544DcF4fcE959C6C4F3b7530190cB5E1BD67Cb7'
```

Comment:
Does this test also work with the old format?

Reply:
Hmm it will create a linkProof but the `account` would follow the new CAIP-10 format not the legacy one.

Reply:
But that's fine I believe because linkProofs which follow the legacy format are handled with the case above ^

Reply:
That sounds right.